### ISSUE TITLE
Add eslint rule to format `TODO` tags and similar meta comments

### ISSUE THREAD

**Issue by dapplion** [ChainSafe/lodestar#5242]
**Is your feature request related to a problem? Please describe.**

For fork items it has been very helpful to tag TODO comments as
```ts
// TODO EIP-4844
```

That way with a text search you can locate pending fork items before the fork

**Describe the solution you'd like**

Add eslint rule to enforce formating

**Comment by maschad** [comment]
Good idea, we could write a [custom rule](https://eslint.org/docs/latest/extend/custom-formatters) but what format what you want to enforce? Something like

```typescript
// TODO: _A summary of the issue_ ${github_issue_hyperlink}  ||   EIP ${EIP-#} ||  FORK ${fork_name}
```

Also would this be a warning or an error?

**Comment by nflaig** [comment]
related https://github.com/ChainSafe/lodestar/pull/5161#discussion_r1124678745

**Comment by wemeetagain** [comment]
@nazarhussain biome rule?
